File: verify_live.py
import requests
import re

def verify():
    urls = [
        "https://trades-avorria-ag-v001.vercel.app/",
        "https://trades-avorria-ag-v001.vercel.app/packages"
    ]
    
    results = {}
    
    # 1. Hero spacing
    r = requests.get(urls[0])
    results['hero_spacing'] = "specialist\ndigital partner for" in r.text or "specialist\n            digital partner for" in r.text or "specialist<br />\n            digital partner for" in r.text or "specialist<br />digital partner for" in r.text
    
    # 2. Stats counters
    results['stats_counters'] = r.text.count('data-counter="500"') > 0
    
    # 3. Trust bar
    results['trust_bar'] = r.text.count("Gas Safe Registered Websites")
    
    # 4. Pricing
    r_pkg = requests.get(urls[1])
    results['price_999'] = r_pkg.text.count("£999") > 0
    results['price_799'] = r_pkg.text.count("£799") > 0
    results['price_1799'] = r_pkg.text.count("£1,799") > 0
    results['price_3499'] = r_pkg.text.count("£3,499") > 0
    
    # 5. Nav
    results['nav_workbench'] = r.text.count("the-workbench") > 0
    
    # 6. Footer
    results['footer_resources'] = r.text.count("Resources") > 0
    
    for k, v in results.items():
        print(f"{k}: {v}")

if __name__ == "__main__":
    verify()

File: verify_live.js
const https = require('https');

function get(url) {
  return new Promise((resolve, reject) => {
    https.get(url, (res) => {
      let data = '';
      res.on('data', (chunk) => { data += chunk; });
      res.on('end', () => { resolve(data); });
    }).on('error', (err) => { reject(err); });
  });
}

async function verify() {
  const home = await get('https://trades-avorria-ag-v001.vercel.app/');
  const packages = await get('https://trades-avorria-ag-v001.vercel.app/packages');

  const results = {
    hero_spacing: home.includes('specialist<br />digital partner for') || home.includes('specialist<br />\n            digital partner for') || home.includes('specialist\n            <br />\n            digital partner for'),
    stats_counters: (home.match(/data-counter="500"/g) || []).length > 0,
    trust_bar_count: (home.match(/Gas Safe Registered Websites/g) || []).length,
    price_999: packages.includes('£999'),
    price_799: packages.includes('£799'),
    price_1799: packages.includes('£1,799'),
    price_3499: packages.includes('£3,499'),
    nav_workbench: home.includes('the-workbench'),
    footer_resources: home.includes('Resources')
  };

  console.log(JSON.stringify(results, null, 2));
}

verify();
